infra/base-images/base-builder/compile_javascript_fuzzer

```diff
@@ -35,6 +35,6 @@ fuzzer_basename=$(basename -s .js $fuzz_target)
 echo "#!/bin/bash
 # LLVMFuzzerTestOneInput so that the wrapper script is recognized as a fuzz target for 'check_build'.
 this_dir=\$(dirname \"\$0\")
-\$this_dir/$project/node_modules/@jazzer.js/core/dist/cli.js $project/$fuzz_target $jazzerjs_args -- \$@" > $OUT/$fuzzer_basename
+\$this_dir/$project/node_modules/@jazzer.js/core/dist/cli.js $project/$fuzz_target $jazzerjs_args \$JAZZERJS_EXTRA_ARGS -- \$@" > $OUT/$fuzzer_basename
```

Comment:
Is `JAZZERJS_EXTRA_ARGS` needed here? I see it only being set in the `base-runner` in the `coverage` script which happens way after this part.

Reply:
Here, `compile_javascript_fuzzer` makes sure that the generated wrapper script accept the `JAZZERJS_EXTRA_ARGS` environment variable to provide extra flags to Jazzer.js. The `coverage` script sets this environment variable before calling the wrapper script, and then the actual replacement with the environment variable take place.

Reply:
got it, thanks!